app-admin/lastpass-binary-component/lastpass-binary-component-4.19.0-r1.ebuild

```diff
@@ -0,0 +1,116 @@
+SRC_URI="https://download.cloud.lastpass.com/linux/lplinux.tar.bz2 -> ${P}.tar.bz2"
+
+LICENSE="LastPass"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="bindist strip mirror" # We can't mirror it, but we can fetch it
+
+LASTPASS_EXEDIR=/opt/lastpass/
+
+QA_PREBUILT="
+	${LASTPASS_EXEDIR}nplastpass*
+"
+
+S="${WORKDIR}"
+
+src_install() {
+	# This is based on the upstream installer script that's in the tarball
+	bin=nplastpass
+	use amd64 && bin="${bin}64"
+	exeinto ${LASTPASS_EXEDIR}
+	doexe "${S}"/${bin}
+
+	# Generate the policy file for Chrome/Chromium/Opera
+	cat >"${T}"/lastpass_policy.json <<-EOF || die
+	{
+		"ExtensionInstallSources": [
+			"https://lastpass.com/*",
+			"https://*.lastpass.com/*",
+			"https://d1jxck0p3rkj0.cloudfront.net/lastpass/*"
+		]
+	}
+	EOF
+	# Install the policy file for Chrome/Chromium/Opera
+	for d in /etc/chromium /etc/opt/chrome; do
+		insinto ${d}/policies/managed
+		doins "${T}"/lastpass_policy.json
+	done
+
+	# Generate the app manifest for Chrome/Opera
+	cat >"${T}"/com.lastpass.nplastpass.json <<-EOF || die
+	{
+		"name": "com.lastpass.nplastpass",
+		"description": "LastPass",
+		"path": "${LASTPASS_EXEDIR}${bin}",
+		"type": "stdio",
+		"allowed_origins": [
+			"chrome-extension://hdokiejnpimakedhajhdlcegeplioahd/",
+			"chrome-extension://debgaelkhoipmbjnhpoblmbacnmmgbeg/",
+			"chrome-extension://hnjalnkldgigidggphhmacmimbdlafdo/",
+			"chrome-extension://hgnkdfamjgnljokmokheijphenjjhkjc/"
+		]
+	}
+	EOF
+	# Install the app manifest for Chrome/Opera
+	# https://developer.chrome.com/apps/nativeMessaging
+	# https://dev.opera.com/extensions/message-passing/
+	insinto /etc/opt/chrome/native-messaging-hosts
+	doins "${T}"/com.lastpass.nplastpass.json
+
+	# Generate the app manifest for Chromium
+	cat >"${T}"/com.lastpass.nplastpass.json <<-EOF || die
+	{
+		"name": "com.lastpass.nplastpass",
+		"description": "LastPass",
+		"path": "${LASTPASS_EXEDIR}${bin}",
+		"type": "stdio",
+		"allowed_origins": [
+			"chrome-extension://hdokiejnpimakedhajhdlcegeplioahd/",
+			"chrome-extension://debgaelkhoipmbjnhpoblmbacnmmgbeg/",
+			"chrome-extension://hgnkdfamjgnljokmokheijphenjjhkjc/"
+		]
+	}
+	EOF
+	# Install the app manifest for Chromium
+	# https://developer.chrome.com/apps/nativeMessaging
+	insinto /etc/chromium/native-messaging-hosts/
+	doins "${T}"/com.lastpass.nplastpass.json
+
+	# Generate the app manifest for Firefox
+	cat >"${T}"/com.lastpass.nplastpass.json <<-EOF || die
+	{
+		"name": "com.lastpass.nplastpass",
+		"description": "LastPass",
+		"path": "${LASTPASS_EXEDIR}${bin}",
+		"type": "stdio",
+		"allowed_extensions": [
+			"support@lastpass.com"
+		]
+	}
+	EOF
+	# Install the app manifest for both firefox and firefox-bin
+	# https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Native_manifests#Manifest_location
+	insinto /usr/lib/mozilla/native-messaging-hosts/
+	doins "${T}"/com.lastpass.nplastpass.json
+	insinto /usr/$(get_libdir)/mozilla/native-messaging-hosts/
```

Comment:
Per https://github.com/mozilla/gecko-dev/blob/b2c8e4842964bc42cd2c6260ac738ab66e379ea5/toolkit/xre/nsXREDirProvider.cpp#L298, the location is either `lib` or `lib64`. `$(get_libdir)` is technically wrong here as it is just a coincidence that it returns `lib64` for amd64 arches. Just hardcode `/usr/lib64` directly.

Reply:
done